xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.cpp

```diff
@@ -1132,9 +1131,14 @@ int CDVDInputStreamNavigator::GetActiveAngle()
   dvdnav_status_t status = m_dll.dvdnav_get_angle_info(m_dvdnav, &current_angle, &number_of_angles);
 
   if (status == DVDNAV_STATUS_OK)
+  {
     return current_angle;
+  }
   else
+  {
+    CLog::LogF(LOGERROR, "Failed to get current angle: {}", m_dll.dvdnav_err_to_string(m_dvdnav));
     return -1;
+  }
```

Comment:
even though this seems not a candidate for warning `control may reach end of non-void function` you can simplify to
```c
if (status == DVDNAV_STATUS_OK)
  return current_angle;

CLog::LogF(LOGERROR, "Failed to get current angle: {}", m_dll.dvdnav_err_to_string(m_dvdnav));
return -1;
```